### PR TITLE
fix: fix vulnerabilities

### PR DESCRIPTION
**Pull request by Razzwan**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,12 +3,11 @@ node_js: 12.18.4
 addons:
   apt:
     update: true
+  chrome: stable
```
Comment (orenyodfat) [inline .travis.yml:6]: why this is needed ?

File: src/settings.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ import BurnerConnectProvider from "@burner-wallet/burner-connect-provider";
 import WalletConnectProvider from "@walletconnect/web3-provider";
 const Torus = require("@toruslabs/torus-embed");
 
+// @TODO: this packages loosk too wired. No description for it at all
```
Comment (orenyodfat) [inline src/settings.ts:10]: these 2 are wallets we are supporting.

File: .travis.yml (hunk 1 above)
Comment (Razzwan) [reply .travis.yml:6]: It was just moved from bottom to top. It is needed for integrational tests. I just fixed `.travis.yml` according to the travis documentation.

File: src/settings.ts (hunk 2 above)
Comment (Razzwan) [reply src/settings.ts]: Removed this comment

File: .travis.yml (hunk 1 above)
Comment (orenyodfat) [reply .travis.yml:6]: though the integration tests works ok without this

Comment (Razzwan) [reply .travis.yml:6]: @orenyodfat no! It was moved from bottom section. Please, see whole file to understand

**Comment by orenyodfat** [comment]
new proposal button does not appears properly https://alchemy-fix-vulnerabili-4vaoqp.herokuapp.com/dao/0x324a1be25798c98516381d523c2d41cd7ce30648/schemes/#
see right top corner..

**Review by orenyodfat — APPROVED** [review]
looks good to me

**Review by roienatan — APPROVED** [review]

**Pull request merged**